public/js/app/components/statistics/Chart.jsx

```diff
@@ -37,17 +45,20 @@ function Charts({ chartNames = [], languageIndex = 1, schools = {} }) {
     <Row>
       {chartNames.map(numberName => (
         <Col key={numberName} xs="4" style={{ paddingRight: 0, paddingLeft: 0 }}>
-          <Chart data={getChartData(numberName, schools)} label={labels[numberName]} />
+          <Chart data={getChartData(numberName, schools)} label={labels[numberName]} languageIndex={languageIndex} />
         </Col>
       ))}
     </Row>
   )
 }
 
-function Chart({ data = [], label = '' }) {
+function Chart({ data = [], label = '', languageIndex }) {
   const styles = {
     font: { fontFamily: 'Open Sans', fontSize: '16px' },
   }
+  const xAxesLable = languageIndex === 0 ? `All\nschools` : `Alla\nskolor`
+  const formatLabel = t => (t === 'ALLS' ? xAxesLable : t)
```

Comment:
All these data has to come in array "data" which is accepted by chart, and it should be dynamic, when users choose only one school it has to hide all schools. So it should be redone in the way all other data comes in "data" array. And languageIndex has to be removed

Reply:
see comment below